Sketch.java

```diff
@@ -1,36 +1,128 @@
 import processing.core.PApplet;
 
+/**
+ * Sketching a house using parameters with methods
+ * @author: B.Zhang
+ */
+
 public class Sketch extends PApplet {
-	
-	
-  /**
-   * Called once at the beginning of execution, put your size all in this method
-   */
   public void settings() {
-	// put your size call here
     size(400, 400);
   }
 
-  /** 
-   * Called once at the beginning of execution.  Add initial set up
-   * values here i.e background, stroke, fill etc.
-   */
   public void setup() {
     background(210, 255, 173);
   }
+ 
+  public void draw() {
+
```

Comment:
missing [level 3 requirement](https://github.com/SACHSTech/ProcessingTask6-CreatingMethods#level-3), "draw a composite object at various locations specified by the method parameters."